src/modules/roc_core/target_posix/roc_core/time.cpp

```diff
@@ -49,6 +52,117 @@ void sleep_for_ms(uint64_t ms) {
         }
     }
 }
+#elif defined(__APPLE__) && defined(__MACH__)
+
+#include <mach/clock.h>
+#include <mach/clock_types.h>
+#include <mach/mach.h>
+#include <mach/mach_time.h>
+
+#define CLOCK_MONOTONIC SYSTEM_CLOCK
```

Comment:
We can remove it, it helps to remember that a xnu use SYSTEM_CLOCK for a monotonic time.